### PR TITLE
[tempo-distributed] fix tempo distributor deployment otlp-http port

### PR DESCRIPTION
**Pull request by Kangkeizai**

### CODE REVIEW

**Review by zanhsieh — CHANGES_REQUESTED** [review]
@Kangkeizai 
Thanks for the contribution. Can you bump the version in Chart.yaml to latest? So later CI can merge this PR once the maintainer approved.

**Comment by Kangkeizai** [comment]
> 

Thanks for the reminder, I've done it.

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**
